connectors/config.py

```diff
@@ -60,9 +64,12 @@ def _default_config():
                 "chunk_size": 1000,
                 "max_concurrency": 5,
                 "chunk_max_mem_size": 5,
+                "max_retries": 5,
```

Comment:
Will need a release note that the default values changed for `max_retries` and `retry_interval`

Reply:
Do you think current release note is okay?

> Release Note
> Improve connection resiliency when accessing elasticsearch - retry all failed calls that fail because of unhealthy Elasticsearch before terminating with an error. This logic is configurable with newly added elasticsearch.max_retries and elasticsearch.retry_interval configuration fields.
> 
> elasticsearch.bulk.max_retries was increased from 3 to 5 to match newly added elasticsearch.max_retries default value. Additionally elasticsearch.bulk.retry_interval was added with default value of 10 seconds (previously was hardcoded to 1 second) to match behaviour of newly added elasticsearch.retry_interval. As a result, bulk inserts will retry for 10 * (1 + 2 + 3 + 4 + 5) = 150 seconds, before this number was 1 * (1 + 2 + 3) = 6 seconds.

Reply:
Ah shit - I wrote this and thought I'd remember to check your release note before hitting submit. And I forgot 🤦